pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -723,6 +723,10 @@
     src = "${hurl.src}/contrib/vim";
   };
 
+  idris2-nvim = super.idris2-nvim.overrideAttrs {
+    dependencies = with self; [ nui-nvim nvim-lspconfig ];
+  };
```

Comment:
could you add 
```
    doInstallCheck = true;
    nvimRequireCheck = "idris2";
```
to check we are not missing anything ? also nui-nvim is said to be optional so I would probably remove it

Reply:
Happy to add those lines to the override. I'll sanity check first, but my recollection is that the plugin fails to start without nui; I assume the README is out of date about it being optional. If I find things do start up without nui I'll remove it!

Reply:
hehe that's why I asked. My goal is to run more and more upstream tests for neovim plugins. 
In absence of those or as a quick hack, the `nvimRequireCheck` is hepful  and already  helped find missing depenencies like that.